recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/wic/prod-devel.wks

```diff
@@ -0,0 +1,11 @@
+# short-description: Create SD card image with a boot partition
+# long-description:
+# Create an image that can be written onto a SD card using dd for use
+# with i.MX SoC family
+# It uses u-boot + other binaries gathered together on imx-boot file
+#
+#part u-boot --source rawcopy --sourceparams="file=imx-boot" --ondisk mmcblk --no-table --align 32
+part / --source rootfs --rootfs-dir=rootfs_boot --ondisk mmcblk --fstype=vfat --label kernel --active --align 16384 --size 32
+part --source rawcopy --sourceparams=file="domd.ext4" --ondisk mmcblk --fstype=ext4 --label domd --align 8192 --size 2G
+
+#bootloader --ptable msdos
```

Comment:
I think this needs to be uncommented. Did you try to mount the resulting wic?

Reply:
here we had a word that it i not necessary , there were an error (i do not remember what was exactly) , but i can uncomment and check the result